test/module/irohad/synchronizer/synchronizer_test.cpp

```diff
@@ -144,7 +148,9 @@ TEST_F(SynchronizerTest, ValidWhenBlockValidationFailure) {
 
   EXPECT_CALL(*mutable_factory, commit_(_)).Times(1);
 
-  EXPECT_CALL(*chain_validator, validateBlock(test_block, _))
+// TODO: 14-02-2018 Alexey Chernyshov replace with expected argument after relocation to
+// shared_model https://soramitsu.atlassian.net/browse/IR-903
+  EXPECT_CALL(*chain_validator, validateBlock(/* testing::Ref(new_test_block) */ _, _))
```

Comment:
I don't really like the idea of test weakening during migration, so maybe this snippet will work?
```cpp
std:unique_ptr<model::Block> old_block(new_test_block.makeOldModel());
EXPECT_CALL(*chain_validator, validateBlock(old_block, _))
```

or more dirtier solution:
```cpp

EXPECT_CALL(*chain_validator, validateBlock(_, _)).InvokeOnce([](auto block, auto) {
  EXPECT_EX(from_old(block), new_test_block);
  return false;
});
```